### DISCUSSION TITLE
nuclei templates  crashing while scanning the target

### DISCUSSION THREAD

**Discussion by Abhinav217-seed** [projectdiscovery/nuclei#3583] [Q&A]
when i was running  nuclei template  for scans facing this error  and the scans were incomplete.

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xea5b86]

goroutine 2922 [running]:
github.com/projectdiscovery/interactsh/pkg/client.(*Client).URL(0x1fb115d?)
        /home/kali/go/pkg/mod/github.com/projectdiscovery/interactsh@v1.1.1-0.20230417162754-2cd861b12467/pkg/client/client.go:569 +0x26
github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/interactsh.(*Client).URL(0xc000301c30?)
        /home/kali/nuclei/v2/pkg/protocols/common/interactsh/interactsh.go:209 +0x8e
github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/interactsh.(*Client).NewURLWithData(0xc000301c30, {0xc0057f399a, 0x12})
        /home/kali/nuclei/v2/pkg/protocols/common/interactsh/interactsh.go:251 +0x2c
github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/interactsh.(*Client).ReplaceWithMarker(0xc00a853c20?, {0xc0057f3980, 0x2c}, 0x4?, {0x37e5178, 0x0, 0x0})
        /home/kali/nuclei/v2/pkg/protocols/common/interactsh/interactsh.go:238 +0x117
github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/interactsh.(*Client).Replace(...)
        /home/kali/nuclei/v2/pkg/protocols/common/interactsh/interactsh.go:232
github.com/projectdiscovery/nuclei/v2/pkg/protocols/http.(*requestGenerator).Make(0xc00816b4f0, {0x2870508, 0xc0086d2930}, 0xc008d89400, {0xc0057f3980?, 0x2c?}, 0xc001fbc570, 0xc001fbc150)
        /home/kali/nuclei/v2/pkg/protocols/http/build_request.go:77 +0xcb
github.com/projectdiscovery/nuclei/v2/pkg/protocols/http.(*Request).ExecuteWithResults.func1({0xc0057f3980, 0x2c}, 0x1?, 0x17?)
        /home/kali/nuclei/v2/pkg/protocols/http/request.go:357 +0x165
github.com/projectdiscovery/nuclei/v2/pkg/protocols/http.(*Request).ExecuteWithResults(0xc0058b4700, 0xc008d89400, 0xc001fbc150, 0xc001fbc240, 0xc001fbc3f0)
        /home/kali/nuclei/v2/pkg/protocols/http/request.go:438 +0x36a
github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/executer.(*Executer).Execute(0xc0058e8f00, 0xc008d89320)
        /home/kali/nuclei/v2/pkg/protocols/common/executer/executer.go:81 +0x383
github.com/projectdiscovery/nuclei/v2/pkg/core.(*Engine).executeTemplateWithTargets.func2.1(0x160036a?, 0x0?, 0xc008d89280)
        /home/kali/nuclei/v2/pkg/core/executors.go:128 +0x296
created by github.com/projectdiscovery/nuclei/v2/pkg/core.(*Engine).executeTemplateWithTargets.func2
        /home/kali/nuclei/v2/pkg/core/executors.go:105 +0x4f8

**Comment by Abhinav217-seed** [comment]
The error message suggests that the problem occurred in the interactsh package  the execution of an HTTP request.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
